### ISSUE TITLE
Mock Api for request context

### ISSUE THREAD

**Issue by gssanthoshbe** [microsoft/playwright#27990]
I am trying to byild an api test for an endpoint that is not ready yet, It would be better if I request.get call also intercept the mock api response

**Comment by dgozman** [comment]
@gssanthoshbe I don't think this is possible today. Also, it seems strange to intercept requests that your test is making, although I understand your usecase. For now, I'd recommend replacing `request.get()` with a hardcoded value.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
